libraries/Microsoft.Bot.Builder.AI.QnA/Utils/QnACardBuilder.cs

```diff
@@ -77,48 +77,82 @@ public static IMessageActivity GetSuggestionsCard(List<string> suggestionsList,
         }
 
         /// <summary>
-        /// Get active learning suggestions card.
+        /// Get Card for MultiTurn scenario. (Can be deprected from 4.10.0 release of sdk).
         /// </summary>
         /// <param name="result">Result to be dispalyed as prompts.</param>
```

Comment:
>dispalyed [](start = 46, length = 9)

Typo

Reply:
Fixed

Reply:
Fixed.

---
In reply to: [433513396](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r433513396) [](ancestors = 433513396)